modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListDisplayContext.java

```diff
@@ -194,6 +230,27 @@ public CPConfigurationEntry getCPConfigurationEntry()
 		return _cpConfigurationEntry;
 	}
 
+	public CreationMenu getCPConfigurationEntryCreationMenu() throws Exception {
+		CPConfigurationList cpConfigurationList = getCPConfigurationList();
+
+		if ((cpConfigurationList != null) && !cpConfigurationList.isMaster() &&
```

Comment:
you should not have also some "permission"? like the update of the catalog or something similar?